### PR TITLE
Add pr template

### PR DESCRIPTION
**Pull request by ttcollins**
This is the backend PR template

### CODE REVIEW

**Comment by ttcollins** [comment]
This is the backend PR template

**Comment by MutegekiHenry** [comment]
@ttcollins Good work, but we dont raise PRs against master branch but rather to the develop-new branch

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+## Checklist:
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] My changes generate no new warnings
+- [ ] Any dependent changes have been merged and published in downstream modules
+
+# Screenshots
```
Comment (MutegekiHenry) [inline .github/pull_request_template.md:31]: @ttcollins I don't think the screenshots section applies for the backend PRs

**Review by MutegekiHenry — COMMENTED** [review]
Good work @ttcollins just a small comment to address

File: .github/pull_request_template.md (hunk 1 above)
Comment (ttcollins) [reply .github/pull_request_template.md:31]: Ohhh, sorry about, totally skipped my mind

**Pull request merged**